packages/moti/package.json

```diff
@@ -34,26 +34,31 @@
   ],
   "exports": {
     ".": {
+      "react-native": "./src/index.tsx",
```

Comment:
This ensures TypeScript can resolve the proper module when type-checking the `./examples/sample` project.